README.md

```diff
@@ -72,6 +72,9 @@ To use one of our SDKs, you must first have a running Meilisearch instance. Cons
 
 After installing `meilisearch-js`, you must import it into your application. There are many ways of doing that depending on your development environment.
 
+> [!WARNING]
+> [Default export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#using_the_default_export) is deprecated and will be removed in a future version. [Issue](https://github.com/meilisearch/meilisearch-js/issues/1789).
+
```

Comment:
Is this fine here? Or should it just be in the release notes? If it's fine should I also mention the `window` deprecation as well here?